test/unit/chips/ChipSet.test.tsx

```diff
@@ -85,7 +85,7 @@ test('#adapter.setSelected adds selectedChipId to state', () => {
   wrapper.instance().adapter.setSelected();
   assert.isTrue(wrapper.state().selectedChipIds.indexOf('1') > -1);
   td.verify(handleSelect(['1']), {times: 1});
-  td.verify(getSelectedChipIds(), {times: 1});
+  assert.equal(1, td.explain(getSelectedChipIds).callCount);
```

Comment:
This seems unrelated to the actual change that you made in top-app-bar. Why did you need to change this?

Reply:
You're right!
ChipSet and TextField code is unrelated with topAppBar.
But you mentioned the warnings. (https://github.com/material-components/material-components-web-react/pull/771#issuecomment-476794882)
So I fix the warning's cause.
Is it better to revert it? And create another PR?

Reply:
Ah I didn't realize this is where the warnings were. I thought this PR had introduced them. Let's leave it. Thanks for fixing it.